File: 4141-count-elements-with-at-least-k-greater-values/count-elements-with-at-least-k-greater-values.java
class Solution {
    public int countElements(int[] nums, int k) {
         if(k<1) return nums.length;
        Arrays.sort(nums);
        int c=0;
        int value=nums[nums.length-k];
        for(int i=0;i<nums.length;i++){
            if(value>nums[i]){
                c++;
            }
        }
        return c;
    }
}

File: 4141-count-elements-with-at-least-k-greater-values/count-elements-with-at-least-k-greater-values.py
class Solution(object):
    def countElements(self, nums, k):
        """
        :type nums: List[int]
        :type k: int
        :rtype: int
        """
        # return len(set(nums))-k if k>1 else len(nums)
        if k<1:
            return len(nums)
        nums.sort()
        c=0
        value=nums[len(nums)-k]
        for x in nums:
            if(value>x):
                 c+=1

        return c
        